pallets/democracy/README.md

```diff
@@ -0,0 +1,141 @@
+# Democracy Pallet
+This repository contains a modified version of Parity's 'democracy' Substrate Pallet. The original version can be found [here](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/democracy).
```

Comment:
Not sure if that is useful for democracy as it might still be an actively maintained branch (used PJs branch for moving to fungibles as base).

Reply:
Exactly, because it is still maintained, it would be more sensible to link to the specific commit/branch/tag we used as a base, rather than the master, which could change and render this README outdated.